control-base/src/main/java/fi/nls/oskari/control/data/GetGeoPointDataHandler.java

```diff
@@ -83,8 +83,12 @@ public void handleAction(final ActionParameters params) throws ActionException {
 
 			final OskariLayer layer = mapLayerService.find(layerId);
 			final String layerType = layer.getType();
-
-			if (OskariLayer.TYPE_WMS.equals(layerType)) {
+			//HSY doesn't want GFI from basemaps.
+			boolean isBaseMap = false;
+			if(layer.isBaseMap() || layer.getAttributes().optBoolean("basemap")) {
+				isBaseMap = true;
```

Comment:
Could this be something like "disableGFI" : [true|false] in attributes instead of tied to basemap? I think using layer.isBaseMap() for this might have some unwanted side-effects